modules/specialized-guides/pages/large-deployments/hub-online-sync.adoc

```diff
@@ -0,0 +1,213 @@
+[[hub-online-sync]]
+= Hub Online Synchronization
+
+
+//OM 2025-04-28: shall we write hub or HUb, peripheral or Peripheral? In the GUI, I saw mixed cases.
+
+== Introduction 
+
+Hub online synchronization reuses the existing repository syncing mechanism and synchronizes channels in the peripheral servers from the repositories on the hub server.
+
+Once the connection between hub and peripheral servers is established, the hub server becomes the main source of data for the peripheral server, effectively replacing {scclongform}.
```

Comment:
>the hub server becomes the main source of data for the peripheral server, effectively replacing {scclongform}.

Not sure if we need to clarify, but this is not limited to vendor channels retrieved from SCC, but it also applies to custom channels. Once they are synced, the peripheral will fetch the packages from the hub and not from the original location of the custom channel defined on the hub.